app/job/rest.py

```diff
@@ -108,6 +108,14 @@ def get_all_notifications_for_service_job(service_id, job_id):
     ), 200
 
 
+@job_blueprint.route('/<job_id>/notification_count', methods=['GET'])
+def get_notification_count_for_job_id(service_id, job_id):
+    count = dao_get_notification_count_for_job_id(job_id)
```

Comment:
we should make sure the service id is valid, the easiest way to do this is to include service_id in the query.

Reply:
If the job_id does not exist the query still returns 0 - is that expected behaviour?

Reply:
Thanks, I pushed those changes now ☺️

Reply:
The api will return 0 if the service or job is not found. That's not really expected behaviour. I've made an update - as I thought it's easier to show than tell.

Reply:
@servingUpAces thanks, only I cannot see the update

Reply:
Ahh, I see, makes sense 👍